tests/README.md

```diff
@@ -0,0 +1,12 @@
+# Backward compatibility and test vector generation
+
+Crate `prio` supports Prio v2, used in [Exposure Notification Private Analytics](enpa-whitepaper), as implemented in [`prio-server`](https://github.com/abetterinternet/prio-server), as well as the Prio v3 Verifiable Distributed Aggregation Function used in [Privacy Preserving Measurements](ppm-spec).
```

Comment:
nit: I prefer breaking lines at 80 characters in markdown files. Feel free to override my preference, of course :)